CRM/Core/Payment/AuthorizeNetIPN.php

```diff
@@ -106,13 +107,6 @@ public function main($component = 'contribute') {
         $objects['contact'] = &$contact;
         $objects['contribution'] = &$contribution;
 
-        // CRM-19478: handle oddity when p=null is set in place of contribution page ID,
```

Comment:
This was copied onto this class as part of https://lab.civicrm.org/dev/financial/-/issues/148 but as noted there it never applied to A.net & was always gonna be short lived on this class - see https://issues.civicrm.org/jira/browse/CRM-19478 for background